src/box/box.cc

```diff
@@ -1354,6 +1362,24 @@ box_process1(struct request *request, box_tuple_t **result)
 	    space_group_id(space) != GROUP_LOCAL &&
 	    box_check_writable() != 0)
 		return -1;
+	if (space_is_memtx(space)) {
+		/*
+		 * Due to on_init_schema triggers set on system spaces,
+		 * we can insert data during recovery to local and temporary
+		 * spaces. However, until recovery is finished, we can't
+		 * check key uniqueness (since index are still not yet built).
+		 * So reject any attempts to write into these spaces.
+		 */
+		if (memtx_space_is_recovering(space)) {
+			diag_set(ClientError, ER_UNSUPPORTED, "Write requests",
```

Comment:
In this case we get very strange message:
2020-12-18 19:39:09.900 [10189] main/103/interactive box.cc:1377 E> ER_UNSUPPORTED: Write requests does not support during snapshot recovery, use box.runtime_info.is_recovery_finished() to check that snapshot recovery was completed

Check ER_UNSUPPORTED error message format